File: PHP/8php 2/stairsIn20.php
<?php
/*
|--------------------------------------------------------------------------
| How many stairs will Suzuki climb in 20 years?
|--------------------------------------------------------------------------
|
*/

function stairsIn20($arr) {
   $res = 0;
   foreach ($arr as $i) {
      $res += array_sum($i);
   }
   return 20 * $res;
}


function stairsIn20($arr) {
   return 20 * array_sum(array_merge(...$arr));
}


function stairsIn20($arr) {
   return 20 * array_reduce($arr, function($acc, $curr) {
     return $acc + array_sum($curr);
   }, 0);
}


function stairsIn20($a) {
   return array_sum(array_map('array_sum', $a)) * 20;
}

File: Python/8py 3/stairs_in_20.py
"""
|--------------------------------------------------------------------------
| How many stairs will Suzuki climb in 20 years?
|--------------------------------------------------------------------------
|
"""

def stairs_in_20(stairs):
   res = 0
   for i in stairs:
      res += sum(i)
   return res * 20


def stairs_in_20(stairs):
   return sum(sum(day) for day in stairs) * 20


def stairs_in_20(stairs):
   return 20 * sum(map(sum, stairs))


def stairs_in_20(stairs):
   return sum(sum(stairs, [])) * 20